test/functional/rpc_decodescript.py

```diff
@@ -133,7 +133,7 @@ def decodescript_script_pub_key(self):
         cltv_script = '63' + push_public_key + 'ad670320a107b17568' + push_public_key + 'ac'
         rpc_result = self.nodes[0].decodescript(cltv_script)
         assert_equal('nonstandard', rpc_result['type'])
-        assert_equal('OP_IF ' + public_key + ' OP_CHECKSIGVERIFY OP_ELSE 500000 OP_CHECKLOCKTIMEVERIFY OP_DROP OP_ENDIF ' + public_key + ' OP_CHECKSIG', rpc_result['asm'])
+        assert_equal('OP_IF ' + public_key + ' OP_CHECKSIGVERIFY OP_ELSE 20a107 OP_CHECKLOCKTIMEVERIFY OP_DROP OP_ENDIF ' + public_key + ' OP_CHECKSIG', rpc_result['asm'])
```

Comment:
This seems much harder to understand to me -- you need to convert it to `0x07a120` then convert that to decimal to get back to the nice round number block height? In particular mistaking it for `0x20a107` (2,138,375) would be particularly misleading.